packages/webcert/src/feature/certificate/Buttons/ForwardCertificateButton.tsx

```diff
@@ -60,7 +58,7 @@ const ForwardCertificateButton: React.FC<Props> = ({
         onClick={handleEmailSend}
         tooltip={description}
         data-testid="forward-certificate-button"
-        startIcon={<FontAwesomeIcon size="lg" icon={faShare} />}>
```

Comment:
The forward arrow points in the wrong direction
Maybe use transform: scaleX(-1)  to reverse it
Also, the vertical alignment looks a bit off

![image](https://user-images.githubusercontent.com/92843575/220853881-87ff3853-bee0-49c3-a65e-49a8f253a0ff.png)

Reply:
Good catch, fixed this. I think the alignment looks a bit better when flipped
![image](https://user-images.githubusercontent.com/28404/220875534-6d2d742b-e6c1-44a7-8f0b-dbd0bc3e0fbf.png)